app/assets/stylesheets/password_resets.scss

```diff
@@ -0,0 +1,48 @@
+h1.reset {
+  padding-top: 100px;
+  font-size: 40px;
+  margin-left: 450px;
+  margin-bottom: 40px;
+  margin-top: 20px;
+  color: chocolate;
+  font-family: "Lato", "Arial", sans-serif;
+  font-weight: bold;
+  align-items: left;
+  white-space: nowrap;
+  text-transform: capitalize;
+}
+
+h1.forgot {
+  padding-top: 100px;
+  font-size: 40px;
+  margin-left: 450px;
+  margin-bottom: 40px;
+  margin-top: 20px;
+  color: chocolate;
+  font-family: "Lato", "Arial", sans-serif;
+  font-weight: bold;
+  align-items: left;
+  white-space: nowrap;
+  text-transform: capitalize;
+}
```

Comment:
You could do this using nested selectors (as you are working with scss), like so:
h1 {
  .reset {
    //code here
  }
  .forgot {
    //code here
  }
}